### DISCUSSION TITLE
Ghostscript?

### DISCUSSION THREAD

**Discussion by Tobshub** [yakovmeister/pdf2image#186] [Q&A]
One of the prerequisites for using this package is ghostscript, but I'm not sure in what way it's used. Could someone please clarify?

**Comment by mskec** [comment]
I believe it's required to run GraphicsMagick on Windows. 

- https://github.com/yakovmeister/pdf2image/blob/master/docs/gm-installation.md#windows
- http://www.graphicsmagick.org/INSTALL-windows.html#prerequisites

Are you able to run the library without ghostscript on Windows?
